databricks/data_access.go

```diff
@@ -346,6 +354,17 @@ func (a *AccessSyncer) syncFilterToTarget(ctx context.Context, do string, aps []
 		return filterName, externalId, err
 	}
 
+	// delete ond row filters
+	for _, ap := range aps {
+		if ap.ActualName != nil {
+			err = sqlClient.DropFunction(ctx, catalogName, schemaName, *ap.ActualName)
```

Comment:
Just to be sure and seems to be looking at the code.

ActualName will contain the "old" function name for a filter and with doing this we will remove it.


This also means that there will be a very small window where both function are active, it's possible to have multiple function attached to data-objects I suspect.

Reply:
Their could be only one function attached as row filter to the table. (If the function is still attached you cannot deleted it).  So by executing the `createOrUpdateRowFilter` method we crate the function and attach it to the table. At that moment the old functions are not attached anymore and could be deleted.

The actualNames of the APs are indeed the old function name. This actually may be only one name. So I think you have a point to check this better.